pkg/expr/sql_command.go

```diff
@@ -111,11 +119,26 @@ func (gr *SQLCommand) Execute(ctx context.Context, now time.Time, vars mathexp.V
 		return rsp, nil
 	}
 
-	rsp.Values = mathexp.Values{
-		mathexp.TableData{Frame: frame},
-	}
+	switch gr.format {
+	case "alerting":
+		numberSet, err := extractNumberSet(frame)
```

Comment:
Kyle described this to me verbally as

> A very specific subset of Multi. Columns must obey a specific order. 

also 

"I don't expect to be calling this function by the time we merge this PR"